Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -73,7 +73,35 @@ open class _ChatChannelListItemView<ExtraData: ExtraDataTypes>: _ChatChannelSwip
         .itemSubviews
         .unreadCountView.init()
         .withoutAutoresizingMaskConstraints
-    
+
+    open var titleText: String? {
```

Comment:
Can you please add documentation?

Reply:
Sure, documentation added.

Reply:
I'm wondering whether this type should be `String` or `String?` - going forward, it would make integration with SwiftUI much easier as most components don't accept optional String. I am not sure if choosing `String` over `String?` will have any significant drawback for UIKit layer (can't think of any right now but maybe I'm missing something)

Reply:
I'd say `String?` is more correct and it allows for better semantic. `""` and `nil` are different things and allow the UI components to behave differently if needed.